package-lock.json

```diff
@@ -2512,6 +2512,35 @@
         "@babel/core": "^7.0.0-0"
       }
     },
+    "node_modules/@babel/plugin-transform-runtime": {
```

Comment:
We just need to clean up the `package-lock` since we no longer need these changes.